tests/integration/cloudformation/api/test_update_stack.py

```diff
@@ -0,0 +1,437 @@
+import json
+import os
+
+import botocore.errorfactory
+import botocore.exceptions
+import pytest
+import yaml
+
+from localstack.utils.files import load_file
+from localstack.utils.strings import short_uid
+from localstack.utils.sync import retry
+
+
+@pytest.mark.aws_validated
+def test_basic_update(cfn_client, deploy_cfn_template, snapshot, is_stack_updated):
```

Comment:
```suggestion
def test_basic_update(cfn_client, deploy_cfn_template, snapshot):
```

also not really a fan of this being a fixture :sweat_smile: